pkg/addon/addon.go

```diff
@@ -1098,3 +1109,95 @@ func FetchAddonRelatedApp(ctx context.Context, cli client.Client, addonName stri
 	}
 	return app, nil
 }
+
+// checkAddonVersionMeetRequired will check the version of cli/ux and kubevela-core-controller whether meet the addon requirement, if not will return an error
+// please notice that this func is for check production environment which vela cli/ux or vela core is officalVersion
+// if version is for test or debug eg: latest/commit-id/branch-name this func will return nil error
+func checkAddonVersionMeetRequired(ctx context.Context, require *SystemRequirements, k8sClient client.Client, dc *discovery.DiscoveryClient) error {
+	if require == nil {
+		return nil
+	}
+
+	// if not semver version, bypass check cli/ux. eg: {branch name/git commit id/UNKNOWN}
+	if version2.IsOfficialKubeVelaVersion(version2.VelaVersion) {
+		res, err := checkSemVer(version2.VelaVersion, require.VelaVersion)
+		if err != nil {
+			return err
+		}
+		if !res {
+			return fmt.Errorf("vela cli/ux version: %s cannot meet require", version2.VelaVersion)
```

Comment:
```suggestion
			return fmt.Errorf("vela cli/ux version: %s cannot meet requirement", version2.VelaVersion)
```

What's the version of UX?

Reply:
also is `version2.version`